### PR TITLE
Add /etc/VERSION mount for Minikube

### PR DESCRIPTION
**Pull request by gnosek**
See https://github.com/draios/sysdig/pull/1600

### CODE REVIEW

File: agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml [hunk 1]
```diff
@@ -25,6 +25,9 @@ spec:
         hostPath:
           path: /etc/os-release
           type: FileOrCreate
+      - name: etc-version
+        hostPath:
+          path: /etc/VERSION
```
Comment (nathan-b) [inline agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml]: I used `type: FileOrCreate` for /etc/os-release above, but if you think /etc/VERSION might also be a directory then maybe that wouldn't be appropriate here

Comment (gnosek) [reply agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml]: /etc/VERSION on the host will either be a file (minikube) or not exist (!minikube). If you mount a nonexistent file anyway, you'll get an empty directory, which is minimally easier for me to detect than a zero-sized file.

I'm happy to do it in any way that makes sense.

**Comment by gnosek** [comment]
Hmm, I think I'd prefer to mount the whole /etc; it's ugly but we won't at least create /etc/VERSION on the host if it doesn't exist.

@nathan-b, thoughts?

**Comment by nathan-b** [comment]
@gnosek for the agent-kmodule container I'd be more open to mounting all of `/etc`, but my feeling is that for the regular agent container that would be too much risk. I wouldn't be surprised to find that many of our customers would balk at this.

If it were necessary to support, say, AWS that would be one thing. But to expose all of our k8s customers to having their `/etc` mapped into the agent container to support the tiny number of people using the agent on Minikube is a bit much, in my opinion.

I would be OK if you checked in the lines but commented out, with instructions to uncomment them if you are running on Minikube.

**Comment by gnosek** [comment]
Yeah, given that Minikube isn't really used in production, I like the idea of commenting out the mounts. Will update the PR.

**Review by mbreitung — APPROVED** [review]

**Pull request merged**